canonical-web-and-design/vanilla-framework
# Borders exploration demo [wip]

## Done

[ Not for review. Just a demo for internal visual design discussion ]

Adjust contrast of borders and creates a noticeable difference between inputs and buttons. Experiments with alternative table headings. Increases space under tables and other block elements using $spv-outer--scaleable.

## QA

- Pull code
- Run `./run serve --watch`
- Open /examples/templates/tick-element-comparison/ to see the demo

Proposal:

![image](https://user-images.githubusercontent.com/2741678/69341525-a2264700-0c61-11ea-9c21-ea4cbc3544fe.png)

Current state:

![image](https://user-images.githubusercontent.com/2741678/69355578-ecb3bd80-0c79-11ea-9f18-9bb45a33427e.png)
